release/src/github.ts

```diff
@@ -83,14 +86,15 @@ export const getMilestoneIssues = async ({
     return [];
   }
 
-  const milestone = await github.rest.issues.listForRepo({
+  // we have to use paginate function or the issues will be truncated to 100
+  const issues = await github.paginate(github.rest.issues.listForRepo, {
```

Comment:
TIL
https://docs.github.com/en/rest/guides/using-pagination-in-the-rest-api?apiVersion=2022-11-28#example-using-the-octokitjs-pagination-method